packages/unigraph-dev-backend/src/custom.d.ts

```diff
@@ -11,4 +11,58 @@ declare global {
 declare type UnigraphUpsert = {
   queries: string[],
   mutations: any[]
-}
\ No newline at end of file
+}
```

Comment:
is fine for now but I think most of this should live in a separate file

I intended for `custom.d.ts` to only be for custom type definitions needed for node modules that don't have any published types

Reply:
I totally agree - I'm thinking of breaking the TS declarations of events into a separate directory/package that can be shared by the frontend and backend. Maybe we can get started once we've made some more progress in frontend?